bin/docbookrx

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env ruby
 
-if File.exist?(app_lib_path = File.join(File.dirname(__FILE__), '../lib/docbookrx'))
+
+if File.exist?(app_lib_path = File.expand_path(File.join(File.dirname(__FILE__), '../lib/docbookrx')))
```

Comment:
Here's the preferred way to write this line that works in both scenarios:

```ruby
if File.exist?(app_lib_path = (File.expand_path '../../lib/docbookrx', __FILE__))
```

Could you use that syntax?

Thanks for pointing this out. Now I get why others have been having trouble executing the scripts out of the development directory.